static/dtale/menu/DataViewerMenuHolder.jsx

```diff
@@ -22,15 +22,17 @@ class ReactDataViewerMenuHolder extends React.Component {
     return (
       <div style={style} className="menu-toggle">
         <div className="crossed">
-          <div
-            className={`grid-menu ${menuOpen ? "open" : ""}`}
-            style={{
-              background: gu.isLight(theme) ? "white" : "black",
-              color: gu.isLight(theme) ? "black" : "white",
-            }}
-            onClick={menuHandler}>
-            <span>&#8227;</span>
-          </div>
+          {menuPinned ? null : (
```

Comment:
```suggestion
          {!menuPinned && (
```

Reply:
Also it looks like your code changes have some formatting issues.  Make sure to run:
* `npm run format`
* `npm run lint -s`

It also looks like you have 4 failing unit tests:
* static/__tests__/reducers/dtale-test.jsx
* static/__tests__/dtale/DataViewer-base-test.jsx
* static/__tests__/iframe/DataViewer-within-iframe-test.jsx
* static/__tests__/iframe/DataViewer-base-test.jsx
* 
Really simple fixes and you can run individual tests using:
`TEST=static/__tests__/iframe/DataViewer-base-test.jsx npm run test-file`